docs/reference/config-options.md

```diff
@@ -48,3 +49,10 @@ IMPORTANT: If you switch this back to `disabled-permanently` from
 `disabled-by-default`, LegalHold devices may still be active in teams
 that have created them while it was allowed.  This may change in the
 future.
+
+### Custom Search Visibility
```

Comment:
I don't understand what "customer search visibility" means, what are the effects when this is enabled or disabled? Who can see what when searching if this is enabled or disabled? Could you please describe this? (also, naming wise, "custom search visibility" doesn't give much idea on what this is about).

Reply:
Hmmm, I guess it's "clear" that this is confusing. This is only meant to allow the team to set a specific value, i.e., one of [these](https://github.com/wireapp/wire-server/blob/custom-search-visibility-limit-name-search/libs/galley-types/src/Galley/Types/Teams/SearchVisibility.hs#L28).

Reply:
Would it be more clear if named something like: `allowChangingSearchVisibility`?

Reply:
Why do we need this feature flag?

Reply:
> Why do we need this feature flag?

I suppose we could just go with making this configurable in every team, and not have a flag that turns this off i `galley.yaml`.  But I think there was a reason for this.  @tiago-loureiro ?